Dockerfile

```diff
@@ -13,7 +13,7 @@ ARG TARGETARCH
 ARG HOST_DIST=$TARGETOS-$TARGETARCH
 
 COPY dist/${HOST_DIST}/synthetic-monitoring-agent /usr/local/bin/synthetic-monitoring-agent
-COPY dist/${HOST_DIST}/k6 /usr/local/bin/k6
+COPY dist/${HOST_DIST}/k6 /usr/local/bin/sm-k6
```

Comment:
We are breaking something with this. I just need to figure out what.

We are not breaking the container, because if we change the default at the same time, I cannot imagine why someone using a container would be providing that value explicitly. Someone using the container in k8s on the other hand, might be passing that value explicitly.

We are not passing a k6 value in the default .service file, so this is why package users are getting errors. We install the required binary as "sm-k6" but the agent expects this to be called "k6". So this change is actually fixing something.

My expectation is that some Dockerfile is going to break, but that's going to manifest as a failure to build the docker image.

Reply:
Ah, this is what this will break:
https://github.com/grafana/sm-k6-runner/blob/8c9a0e723cf3c2d3fda16c3d697b0454f9b203d9/Dockerfile#L24

We'll fix it when we get there tho.